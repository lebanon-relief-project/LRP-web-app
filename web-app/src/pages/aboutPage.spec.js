import React from "react";
import { render } from "@testing-library/react";
import AboutPage from "./aboutPage";
import "jest-styled-components";

jest.mock("../components/HelpSection", (props) => {
  return (props) => <div>Help section</div>;
});

describe("the AboutPage component", () => {
  it("should match the snapshot", () => {
    const { container } = render(<AboutPage />);
    expect(container).toMatchSnapshot();
  });
});
