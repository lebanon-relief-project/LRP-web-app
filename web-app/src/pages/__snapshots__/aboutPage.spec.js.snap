// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the AboutPage component should match the snapshot 1`] = `
.c5 {
  display: block;
  font-family: Playfair Display;
  font-style: normal;
  font-weight: 900;
  font-size: 38px;
  line-height: 46px;
  color: #000000;
}

.c7 {
  position: relative;
  margin: 0;
  font-family: Raleway;
  font-style: normal;
  font-weight: 400;
  font-size: 16px;
  line-height: 24px;
  color: #000000;
  z-index: 1;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c6 {
  padding-top: 10%;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0;
  width: calc(100vw);
  margin-left: 50%;
  -webkit-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
}

.c1 {
  max-width: 1440px;
  margin-top: 80px;
  padding: 0 240px;
}

.c4 {
  position: relative;
  width: 50%;
}

.c3 {
  object-fit: contain;
  margin: 2%;
  max-width: 60%;
}

.c9 {
  heigth: auto;
  width: 10%;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.c8 {
  background: inherit;
  margin: 0;
  padding: 0;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (max-width:576px) {
  .c2 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:576px) {
  .c6 {
    padding-top: 0;
  }
}

@media (max-width:1025px) {
  .c1 {
    padding: 0 18px;
  }
}

@media (max-width:576px) {
  .c4 {
    right: 0;
    width: 100%;
    -webkit-order: 2;
    -ms-flex-order: 2;
    order: 2;
  }
}

@media (max-width:768px) {
  .c3 {
    width: 80%;
  }
}

@media (max-width:576px) {
  .c3 {
    width: 100%;
    max-width: 100%;
    padding-left: 0rem;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <img
          alt="City"
          class="c3"
          src="City.svg"
        />
        <div
          class="c4"
        >
          <h2
            class="c5 c6"
          >
            About us
          </h2>
          <p
            class="c7"
          >
            Following the devastating explosion of Beirutâ€™s port in August 2020, Lebanon Relief Network was founded by six individuals who themselves were impacted by the incident and experienced signs of trauma. The founding members observed a lack of resources and toolsfor the Lebanese community to use to recover from trauma.
          </p>
        </div>
      </div>
      <div
        class="c2"
      >
        <div
          class="c4"
        >
          <p
            class="c7"
          >
            Joined by volunteers across the world, they worked to fill this gap, educate the community, and aid in advancing mental well-being. Today, Lebanon Relief Network encompasses a team of over 50 volunteers spread all over the world who collectively possess a wide range of experiences in combating trauma. 
            <br />
             
            <br />
             We are continuously building valuable partnerships with volunteers and mental health professionals to create a powerful network for support and healing. Through our digital platform, we produce content informed and validated by experts to offer a safe space for healing, offer tools and resources for self-help, and foster trauma-related dialogue. The enthusiasm, commitment, and resilience of our team is at the core of what Lebanon Relief Network stands for.
            <br />
             
            <br />
             
            <b>
              We are LRN.
            </b>
          </p>
        </div>
        <img
          alt="Map"
          class="c3"
          src="Map.svg"
        />
      </div>
    </div>
  </div>
  <section
    class="c8"
  >
    <img
      alt="loading..."
      class="c9"
      src="LoadingSpinner.gif"
    />
  </section>
  <div>
    Help section
  </div>
</div>
`;
